.flake8

```diff
@@ -1,7 +1,7 @@
 [flake8]
 inline-quotes = "
 ignore = E203, E266, W503, ANN002, ANN003, ANN101, ANN102, ANN401, N807, N818
-max-line-length = 79
+max-line-length = 96
```

Comment:
In this file there are flake8 rules that Python developers should follow, if you made a mistake you need to fix your code not this file, return max-line-length=79